README.md

```diff
@@ -1,3 +1,5 @@
+⚠️ This repository has been archived. It is no longer actively maintained and will only remain accessible in a read-only mode.
```

Comment:
I usually put the date in the note - eg 'As of Feb 2023...'

Reply:
added